packages/client/tsconfig.json

```diff
@@ -20,6 +20,9 @@
     "jsx": "preserve",
     "downlevelIteration": true
   },
+  "references": [
+    { "path": "../common" }
+  ],
```

Comment:
The `references` sections in the client and main package's tsconfigs causes the common package to automatically build when running `tsc --build` or `tsc -b`. For more on that Typescript feature see https://www.typescriptlang.org/docs/handbook/project-references.html